core-services/release-controller/_repos/ocp-4.11-rhel8.repo

```diff
@@ -34,6 +34,18 @@ sslclientkey = /tmp/key/rh-cdn.pem
 sslclientcert = /tmp/key/rh-cdn.pem
 failovermethod = priority
 
+[rhel-8-nfv]
+name = rhel-8-nfv
+baseurl = https://cdn.redhat.com/content/tus/rhel8/8.6/x86_64/nfv/os/
```

Comment:
Are you sure we don't need to point `/rhel8/8/` instead of `/rhel8/8.6`? this is exactly the fix that ART had to apply recently. (they are using the `pulp` repo though).

Reply:
That's the root of the issue. Those should be CDN backed repos which are correctly populated for each versions. The other URLs are Pulp filters which do not get updated correctly for some weird reasons.

Reply:
See all the URLs for all those repos